sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_data_lake_directory_client.py

```diff
@@ -133,6 +133,32 @@ def create_directory(self, metadata=None,  # type: Optional[Dict[str, str]]
             For example, if p is 0777 and u is 0057, then the resulting permission is 0720.
             The default permission is 0777 for a directory and 0666 for a file. The default umask is 0027.
             The umask must be specified in 4-digit octal notation (e.g. 0766).
+        :keyword str owner:
+            The owner of the file or directory.
+        :keyword str group:
+            The owning group of the file or directory.
+        :keyword str acl:
+            Sets POSIX access control rights on files and directories. The value is a
+            comma-separated list of access control entries. Each access control entry (ACE) consists of a
+            scope, a type, a user or group identifier, and permissions in the format
+            "[scope:][type]:[id]:[permissions]".
+        :keyword str lease_id:
+            Proposed lease ID, in a GUID string format. The DataLake service returns
+            400 (Invalid request) if the proposed lease ID is not in the correct format.
+        :keyword int lease_duration:
+            Specifies the duration of the lease, in seconds, or negative one
+            (-1) for a lease that never expires. A non-infinite lease can be
+            between 15 and 60 seconds. A lease duration cannot be changed
+            using renew or change. Default is -1 (infinite lease).
+        :keyword expiry_options:
+            Indicates mode of the expiry time.
+            Possible values include: 'NeverExpire', 'RelativeToNow', 'Absolute'"
+        :paramtype expiry_options: str or Literal["NeverExpire", "RelativeToNow", "Absolute"]
+        :keyword expires_on:
```

Comment:
Sorry I should have caught this earlier but if `create_directory` does not support expiry we should not include them in the docstring.